cmd/kubeadm/app/util/version.go

```diff
@@ -83,3 +91,33 @@ func KubernetesVersionToImageTag(version string) string {
 	allowed := regexp.MustCompile(`[^-a-zA-Z0-9_\.]`)
 	return allowed.ReplaceAllString(version, "_")
 }
+
+// KubernetesValidateVersion is a helper function that can validate
+// if user requested release version or version from CI build area
+// and then return validated and/or resolved from keywords like "latest".
+//
+// Examples of supported versions:
+// v1.7.1 -- released version.
+// release/v1.7.1 -- same as above.
+// stable-1.7 or release/stable-1.7 -- labeled official releases
+// ci/v1.8.0-alpha.1.123+012345678 -- CI explicit build version
+// ci/latest-1.8 or ci-cross/latest-1.8 -- CI labeled builds
```

Comment:
I don't think we should confuse the user with `ci-cross` although we fetch the images from there under the hood.
Let's not support for `ci-cross/` at the moment.